bin/options.js

```diff
@@ -21,6 +25,31 @@ module.exports = getOptions;
  */
 const defaultPathname = 'test/mocha.opts';
```

Comment:
```js
const defaultPathname = path.join(getTestDirectory(), DEFAULT_OPTS_FILENAME);
```

Reply:
I did several other edits that have vanished. The biggest was this change:
`const pkg = readPkgUp.sync('__dirname').pkg;`